gateway/libexec/boot.lua

```diff
@@ -1,3 +1,8 @@
+-- Clean warning on openresty 1.15.8.1, where some global variables are set,
+-- and a warning message is show during startup outside apicast packages.
+-- Code related: https://github.com/openresty/lua-nginx-module/blob/61e4d0aac8974b8fad1b5b93d0d3d694d257d328/src/ngx_http_lua_util.c#L795-L839
+getmetatable(_G).__newindex = nil
```

Comment:
This will crash on OpenResty < 1.15. But I guess that is not that big of a deal.

Reply:
It's this is the only incompatibility we have introduced with OpenResty 1.13.x. It might be worth to use `(getmetatable(_G) or {})` instead like in lua-rover.